### PR TITLE
LRQA-53640

### PR DESCRIPTION
**Pull request by Adriverforlife**
This will satisfy the requirements for LRQA-53642 s well.

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase [hunk 1]
```diff
@@ -666,6 +666,112 @@ definition {
...
+
+		AssertElementPresent(
+			key_avgTime = "0min",
+			locator1 = "WorkflowMetrics#METRICS_PERFORMANCE_BY_STEP_ROW_CHECK",
+			rowNumber = "2");
+
+		AssertElementPresent(
+			key_avgTime = "1min",
+			locator1 = "WorkflowMetrics#METRICS_PERFORMANCE_BY_STEP_ROW_CHECK",
+			rowNumber = "1");
```
Comment (barrosdennys) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase]: Hi @Adriverforlife 
please check WorkflowMetricsPerformanceByStep.macro file

I believe that viewPerformanceByStepValues can be used to validate these values...

Would you please check it?

Comment (Adriverforlife) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase]: That macro checks locations on a different page. It comes up with "element not found" on the page I have it looking at for this test.

**Comment by barrosdennys** [comment]
Just started reviewing :)

**Comment by barrosdennys** [comment]
Hey @Adriverforlife ,
I reviewed the PR again, and it is all fine. Thank you very much!

**Comment by barrosdennys** [comment]
Sent to Bchan
https://github.com/barrosdennys/liferay-portal/pull/89

**Pull request closed without merging**